apiCalls.js

```diff
@@ -14,7 +14,7 @@ export const getActivities = (lat, long) => {
 
 export const getAllTrips = () => {
   const query = {
-      "query": "{allTrips(userApiKey: \"b9aead4b955bccb5c57ef830580f3de5\") {id name origin originAbbrev originLat originLong tripdestinationSet {destination {location abbrev lat long} startDate endDate activitySet {name date category}}}}"
+      "query": "{allTrips(userApiKey: \"b9aead4b955bccb5c57ef830580f3de5\") {id name origin originAbbrev originLat originLong tripdestinationSet {destination {location abbrev lat long} id startDate endDate activitySet {name date address category rating image lat long tripDestination { trip { name } destination { location abbrev } } } } } }"
```

Comment:
Still not sure whats happening here.

Reply:
A general overview:

Fetching from Graphql requires some adjustments to our typical RESTFUL fetching. All fetches use the POST method, and fall into two categories of queries (get data) and mutations (update, add, or delete data).

The second 'argument' of properties within the curlies (after the API key) tell the endpoint what to specifically return in the fetch response